shared/tests/utility/math/docs/README.txt

```diff
@@ -0,0 +1,14 @@
+The files in this folder contain the same input data used in TestCoordinates.cpp along with some matlab scripts that perform the same calculations found in coordinates.h
```

Comment:
Good call! I'll do that.

Reply:
If you want to see what markdown is supported within here you should look at github flavoured markdown, as that's what it will be rendered with https://github.github.com/gfm/